sprintQuovantis/sprintParam/views.py

```diff
@@ -103,3 +106,76 @@ def post(self, request):
                     "message": "Data not stored"
                 },
                 status=status.HTTP_400_BAD_REQUEST)
+
+
+class UserVote(APIView):
+    """
+    This class is creating for vote records
+    """
+    def post(self, request):
+        """
+        this method is created for inserting the voting data
+        :param request: format of the request
+        :return: Response
+        """
+        try:
+            serializer = VoteSerializer(data=request.data)
+            serializer.is_valid(raise_exception=True)
+            serializer.save()
+            return Response(
+                {
+                    "Message": "User Vote Successfully",
+                    "data": serializer.data
+                },
+                status=status.HTTP_201_CREATED
+            )
+        except ValidationError:
+            logging.error("Validation Failed")
+            return Response(
+                {
+                    "Message": "Validation Failed"
+                },
+                status=status.HTTP_400_BAD_REQUEST
+            )
+        except Exception as e:
+            logging.error(e)
+            return Response(
+                {
+                    "Message": "Invalid User"
+                },
+                status=status.HTTP_400_BAD_REQUEST
+            )
+
+
+class Result(APIView):
+    """
+    This class Using for Voting Result
+    """
+
+    def get(self, request):
+        """
+        this method is created for fetching the voting data
+        :param request: format of the request
+        :return: Response
+        """
+        try:
+            votes = Votes.objects.filter(vote_for=request.data.get("vote_for"))
+            for i in votes:
+                print(i.parameter_id)
+
+            return Response(
+                {
+                    "message": "Total votes ",
+                    "Votes": len(list(i for i in votes))
+                },
+                status=status.HTTP_201_CREATED)
```

Comment:
GET requests shiould not return "201 created" as http response code. "201 created" is only meant for when you create a new resource.